LayoutTests/imported/w3c/web-platform-tests/wai-aria/role/menu-roles-expected.txt

```diff
@@ -0,0 +1,43 @@
+Tests menu and related roles.
+
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+x
+
+PASS role is menu
+PASS role is menuitem (in menu)
+PASS role is group (in menu)
+PASS role is menuitem (in group, in menu)
+PASS role is menuitemradio (in group, in menu)
+PASS role is menuitemcheckbox (in group, in menu)
+PASS role is menubar
+PASS role is menuitem (in menubar)
+PASS role is group (in menubar)
+PASS role is menuitem (in group, in menubar)
```

Comment:
https://wpt.fyi/results/wai-aria/role/menu-roles.html
PASS role is menuitem (in group, in menu)
PASS role is menuitem (in group, in menubar)
Two tests pass after this commit.